Sort arcs by story order instead real date

derpynirvash opened the pull request:
Some basic code to allow arcs to be sorted by a value, time_line in the header of each file.

irisfofs commented on .themes/default/source/arcs.html:
```diff
@@ -15,8 +15,6 @@ <h3 class="arc-title"><a name="{{ rp.name | slugify }}" href="#{{ rp.name | slug
     {% endfor %}
     </ul>
-  {% else %}
-    {% include rp.html rp=rp %}
```
Why remove this? Now RPs not part of an arc won't be displayed on the arc page.

derpynirvash replied:
It is an index of all arcs, why would you put non arc'ed rps on the page? Leads to a lot of clutter.

irisfofs replied:
Well, it's more of a display that displays arcs when they exist, and also other ones. I dunno, haven't really thought about it.

I'd rather not change it now as part of this, that should be its own thing.

derpynirvash replied:
Eh, in my opinion, it at least needs to be sorted better.

But sounds fine, leaving design decisions for later.

irisfofs replied:
Can you make an issue for the Arc View sorting being confusing / needing to
be improved?

On Wed, Jul 8, 2015, 15:25 derpynirvash notifications@github.com wrote:

> In .themes/default/source/arcs.html
> https://github.com/xiagu/jekyll-rp_logs/pull/19#discussion_r34188202:
> 
> > @@ -15,8 +15,6 @@ <h3 class="arc-title"><a name="{{ rp.name | slugify }}" href="#{{ rp.name | slug
> >        {% include rp.html rp=arc_rp %}
> >      {% endfor %}
> >      </ul>
> > -  {% else %}
> > -    {% include rp.html rp=rp %}
> 
> Eh, in my opinion, it at least needs to be sorted better.
> 
> But sounds fine, leaving design decisions for later.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/xiagu/jekyll-rp_logs/pull/19/files#r34188202.

derpynirvash commented:
Style stuff was like, removed, man.

xiagu commented:
Yeah, sorry, I was at a con. :P I want to do a bit of tweaking to stuff once I merge it, so I have held off until I'm ready to do that.

derpynirvash committed:
Updated time_line implementation and removed debug from skype12

Removed debug prints from skype 12 hour parser

Changed time_line implementation so that arcs will be sorted normally, unless the time_line value is set inside a file, if it is, the time_line value overrides the normal start rp time

derpynirvash commented:
As requested, change the way that the arcs get sorted so that the timeline value will just override the start date of the file.

xiagu commented:
(thought it was actually merged, just... more complicatedly.)
